l10n_it_fatturapa_in/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 
 {
     'name': 'Italian Localization - Fattura elettronica - Ricezione',
-    'version': '11.0.1.0.0',
+    'version': '11.0.1.1.0',
```

Comment:
@eLBati  Facendo il merge ho un conflitto.  l'attuale ha versione  '11.0.1.0.1'
come mai hai aumentato la penultima e non l'ultima?

Reply:
@labaggio grazie, conflitti risolti.

> come mai hai aumentato la penultima e non l'ultima?

In base a https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers